includes/admin/class-wc-admin-setup-wizard.php

```diff
@@ -393,6 +431,13 @@ public function setup_wizard_steps() {
 			unset( $output_steps['activate'] );
 		}
 
+		unset( $output_steps['new_onboarding'] );
+
+		// Don't show the navigation on the onboarding prompt screen.
+		if ( 'new_onboarding' === $this->step ) {
```

Comment:
Optional: as an alternative, we could use a body class (something like `wc-setup__new-onboarding`) to hide various elements instead of adding this extra logic to each template pieces.

Reply:
Handled in https://github.com/woocommerce/woocommerce/pull/24991/commits/1fabf2f14582bdf04717422075e9d48744051b81.